server/api/dataset.py

```diff
@@ -248,6 +250,33 @@ def submitReviewImages(self, dataset, params):
         return {'status': 'success'}
 
     @describeRoute(
+        Description('Get registered metadata for a dataset.')
+        .param('id', 'The ID of the dataset.', paramType='path')
+    )
+    @access.user
+    @loadmodel(model='dataset', plugin='isic_archive', level=AccessType.READ)
+    def getRegisteredMetadata(self, dataset, params):
+        File = self.model('file')
+        User = self.model('user', 'isic_archive')
+
+        user = self.getCurrentUser()
+        User.requireCreateDataset(user)
+
+        return [
+            {
+                'file': File.load(
+                    registration['fileId'], force=True, exc=True,
+                    fields=['_id', 'name']),
```

Comment:
Specifying `fields` causes an error in the Girder version we use:
```
  File "/home/vagrant/girder/girder/utility/acl_mixin.py", line 63, in load
    loadType = doc['attachedToType']
KeyError: 'attachedToType'
```
To fix it I think we'd need https://github.com/girder/girder/commit/936b0ba9ea10264e5a6f0bae1516d804fd061cfd and perhaps the followup commit https://github.com/girder/girder/commit/8b10a0807214a14319682e9c6f9f809f7ccac538.